package.json

```diff
@@ -1,24 +1,25 @@
 {
```

Comment:
The key changes look good, and some I'd like clarification on. What's the motivation behind the following changes:
- bugs
- directories
- repository url
- dependencies
- keywords